### ISSUE TITLE
High Sierra: Unable to open the database Wrong key or database file is corrupt

### ISSUE THREAD

**Issue by frranck** [keepassxreboot/keepassxc#1913]
Can't open the database.

## Current Behavior

Can't open the database: 
"Unable to open the database Wrong key or database file is corrupt"

## Steps to Reproduce (for bugs)

1. select keyfile
2. enter password
3.
4.

## Context

Worked only once (!) since I updated to OSX High Sierra.
Same database/password works fine on android.

## Debug Info
KeePassXC - Version 2.3.1
Revision: 2fcaeea

Libraries:
- Qt 5.9.3
- libgcrypt 1.8.2

Operating system: macOS High Sierra (10.13)
CPU architecture: x86_64
Kernel: darwin 17.5.0

Enabled extensions:
- Auto-Type
- Browser Integration
- Legacy Browser Integration (KeePassHTTP)
- SSH Agent
- YubiKey

**Comment by droidmonkey** [comment]
Try a fresh transfer from your Android to your Mac again.

**Comment by frranck** [comment]
@droidmonkey  Hi, so I copied the file with a different name on the mac, copied it on my android, opened it. It's working. 
When I try to open the new file on the mac it's still failing with the error.

No Unicode characters.

**Comment by droidmonkey** [comment]
I can only think of things like caps lock being on or foreign keyboard layout causing issues.

**Comment by frranck** [comment]
Yeah but I'm clicking the little eye to verify the password.

**Comment by frranck** [comment]
Also If I create a new database with a super simple password, then close keepassxc then try to open it again it fails.

**Comment by droidmonkey** [comment]
Are you saving the database locally or to some form of cloud/network storage? I hate to give this advice, but have you tried reinstalling KeePassXC? This is not normal behavior and obviously does not impact our general audience (ie, something is up with your setup)

**Comment by frranck** [comment]
Locally. It's a brand new install, I used to use KeepassX that had the same issue since I upgraded to High Sierra. Then I tried KeePassXC that could open my previous database. But it worked only once.

**Comment by frranck** [comment]
I tried with a new version from the .DMG instead of my brew version and it still doesn't work.

**Comment by frranck** [comment]
Also the strange thing is that there's nothing  weird with my setup. I don't have any issue with any other  software...

**Comment by weslly** [comment]
Can you open it with from `keepassxc-cli`?

**Comment by frranck** [comment]
I don't have it installed with the brew package. where is it ?

**Comment by phoerious** [comment]
Due to a packaging bug, keepasxc-cli is not part of the DMG. We need to fix that for 2.3.4.

**Comment by frranck** [comment]
Hello, I just installed the latest DMG, how do I access the keepasxc-cli ?

**Comment by phoerious** [comment]
It's not contained in the DMG, unfortunately. It's a bug which has been fixed in #2165, but unfortunately, we didn't get that fix ready in time for 2.3.4.

**Comment by frranck** [comment]
Hello, seems my database file was simply corrupted.
Thanks for your help, will close this issue.
